modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryLocalServiceImpl.java

```diff
@@ -28,4 +32,38 @@
 )
 public class AccountEntryLocalServiceImpl
 	extends AccountEntryLocalServiceBaseImpl {
+
+	@Override
+	public AccountEntry addAccountEntry(
+			long parentAccountEntryId, String name, String description,
+			long logoId, int status, ServiceContext serviceContext)
```

Comment:
Is it necessary to require a ServiceContext for this one? Can we just ask for a `userId` instead?

Also, I think I'd prefer to move `parentAccountEntryId` after `name` and `description`.

Reply:
It's not required to have `ServiceContext`. I added it because I see it being included in the method signatures for users and orgs. I can replace it with `userId`.

And for the ordering, I follow the order it's defined in the service.xml. That's also the ordering for `addOrganization` https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portal/service/impl/OrganizationLocalServiceImpl.java#L239-L242. I don't have a preference as long as it passes BChan's review :)

Reply:
Let's go ahead and change the order of parameters. Fine by me if we also change the order in the `service.xml` file, that way it will stay consistent :)

Reply:
👍

I will update the parameter order. I don't think we need to update the order in the `service.xml` since BChan already did (https://github.com/liferay/liferay-portal/commit/e1f1e7eaba6e114ba2b8b182761fd87812f21c5e).